snapcraft/cli/store.py

```diff
@@ -76,10 +77,14 @@ def _human_readable_acls(store: storeapi.StoreClient) -> str:
         if not acl[key]:
             human_readable_acl[key] = 'No restriction'
 
+    expiration_date = parser.parse(str(acl['expires']))
```

Comment:
can't this be solve with `datetime.strptime` like in `snapcraft.cli.assertions`?

Reply:
strptime [doesn't handle 8601 timezones](https://stackoverflow.com/questions/969285/how-do-i-translate-a-iso-8601-datetime-string-into-a-python-datetime-object).